src/dbus.c

```diff
@@ -588,8 +588,8 @@ GVariant *dbus_cb_dunst_Properties_Get(GDBusConnection *connection,
 {
         struct dunst_status status = dunst_status_get();
 
-        if (STR_EQ(property_name, "running")) {
-                return g_variant_new_boolean(status.running);
+        if (STR_EQ(property_name, "paused")) {
+                return !g_variant_new_boolean(status.running);
```

Comment:
You're inverting the return pointer, not the boolean value. This should be
```c
                 return g_variant_new_boolean(!status.running); 
```

Reply:
Sorry my bad. Fixed below

Reply:
I see the fix on the XML but not this, perhaps you forgot to commit it?

Reply:
Yep it was still unstaged..